WcaOnRails/app/views/committees/show.html.erb

```diff
@@ -0,0 +1,79 @@
+        <colgroup>
+          <col />
+          <col class="col-md-4" />
+          <col class="col-md-4" />
+          <col class="col-md-4" />
+        </colgroup>
+        <thead>
+          <tr>
+            <th></th>
+            <th>Name</th>
+            <th>Start Date</th>
+            <th>Role</th>
+          </tr>
+        </thead>
+        <tbody>
+        <% team.current_members.includes(:user).includes(:committee_position).order("committee_positions.team_leader desc").order("users.name asc").each do |member| %>
+          <tr class="<%= member.committee_position.slug %>">
+            <td>
+              <% if @committee.slug == Committee::WCA_DELEGATES_COMMITTEE %>
+                <%= mail_to member.user.email, target: "_blank", class: "hide-new-window-icon" do %>
+                  <span class="glyphicon glyphicon-envelope"></span>
+                <% end %>
+              <% end %>
+              <% if current_user&.can_edit_users? %>
+                <%= link_to edit_committee_team_team_member_path(@committee, team, member) do %>
+                  <span class="glyphicon glyphicon-pencil"></span>
+                <% end %>
+              <% end %>
+            </td>
+            <td>
+              <%= link_to "/results/p.php?i=#{member.user.wca_id}", class: "wca-user-link" do %>
+                <%= render "shared/user", user: member.user %>
+              <% end %>
+            </td>
+            <td><%= member.start_date.strftime('%B %d, %Y')  %></td>
+            <td><%= member.committee_position.name %></td>
+          </tr>
+        <% end %>
+        </tbody>
+      </table>
+    </div>
+    <% if current_user&.can_manage_committees? %>
+      <p>
+        <%= link_to icon("edit", "Edit #{team.name}"), edit_committee_team_path(@committee, team), class: "btn btn-info" %>
+        <%= link_to icon("user-plus", "Add a member to #{team.name}"), new_committee_team_team_member_path(@committee, team), class: "btn btn-success" %>
+        <% if team.team_members.empty? %>
+          <%= link_to icon("trash", "Delete #{team.name}"), committee_team_path(@committee, team), method: :delete, data: { confirm: 'Are you sure you want to delete this team?' }, class: "btn btn-danger" %>
+        <% end %>
+      </p>
+      <hr />
+    <% end %>
+  <% end %>
+  <p>
+    <%= link_to icon("undo", "Back to Committees"), committees_path, class: "btn btn-primary" %>
+    <% if current_user&.can_manage_committees? %>
+      <%= link_to icon("edit", "Edit Committee"), edit_committee_path, class: "btn btn-info" %>
+      <% if @committee.teams.empty? %>
+        <%= link_to icon("trash", "Delete Committee"), committee_path(@committee), method: :delete, data: { confirm: 'Are you sure you want to delete this committee?' }, class: "btn btn-danger" %>
+      <% end %>
+      <%= link_to icon("plus", "Create New Team"), new_committee_team_path(@committee), class: "btn btn-success" %>
+    <% end %>
+  </p>
+</div>
```

Comment:
If I'm right that this page works as the public '/delegates' one, it should be internationalized.